packages/contracts/deployments/engine/V3/partners/tender/MINTER_DEPLOYMENT_LOGS.log

```diff
@@ -0,0 +1,82 @@
+    at Logger.makeError (/Users/jakerockland/Code/artblocks-contracts/node_modules/@ethersproject/logger/src.ts/index.ts:269:28)
+    at Logger.throwError (/Users/jakerockland/Code/artblocks-contracts/node_modules/@ethersproject/logger/src.ts/index.ts:281:20)
+    at checkError (/Users/jakerockland/Code/artblocks-contracts/node_modules/@ethersproject/providers/src.ts/json-rpc-provider.ts:78:20)
+    at EthersProviderWrapper.<anonymous> (/Users/jakerockland/Code/artblocks-contracts/node_modules/@ethersproject/providers/src.ts/json-rpc-provider.ts:642:20)
+    at step (/Users/jakerockland/Code/artblocks-contracts/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:48:23)
+    at Object.throw (/Users/jakerockland/Code/artblocks-contracts/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:29:53)
+    at rejected (/Users/jakerockland/Code/artblocks-contracts/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:21:65)
+    at processTicksAndRejections (node:internal/process/task_queues:95:5) {
+  reason: 'execution reverted: Only Core AdminACL allowed',
+  code: 'UNPREDICTABLE_GAS_LIMIT',
+  method: 'estimateGas',
+  transaction: {
+    from: '0xB8559AF91377e5BaB052A4E9a5088cB65a9a4d63',
+    to: '0xf377a477704DD68907BC603A7403B09B7b97DC0E',
+    data: '0xbee4d1060000000000000000000000000283873c59ef310359a82a446508414f3b8d9c90',
+    accessList: null
+  },
+  error: ProviderError: execution reverted: Only Core AdminACL allowed
+      at HttpProvider.request (/Users/jakerockland/Code/artblocks-contracts/node_modules/hardhat/src/internal/core/providers/http.ts:88:21)
+      at processTicksAndRejections (node:internal/process/task_queues:95:5)
+      at EthersProviderWrapper.send (/Users/jakerockland/Code/artblocks-contracts/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)
+}
+[ERROR] unable to allowlist the new minter on the minter filter at 0xf377a477704DD68907BC603A7403B09B7b97DC0E
+[INFO] this may be because the minter filter has already been transferred to a partner's wallet
+[INFO] Verifying MinterDAExpV4 contract deployment...
+The contract 0x0283873C59ef310359a82a446508414F3b8D9C90 has already been verified
+[INFO] MinterDAExpV4 contract verified on Etherscan at 0x0283873C59ef310359a82a446508414F3b8D9C90}
+[INFO] Minter deployment details written/appended to /Users/jakerockland/Code/artblocks-contracts/packages/contracts/deployments/engine/V3/partners/tender/DEPLOYMENTS.md
+[INFO] Completed minter deployment! MinterDAExpV4 deployed to 0x0283873C59ef310359a82a446508414F3b8D9C90, and allowlisted on the minter filter at 0xf377a477704DD68907BC603A7403B09B7b97DC0E
+[ACTION] If this minter is to be indexed in a subgraph, ensure it is added to the subgraph's config.
+[ACTION] The minter was not able to be allowlisted on the minter filter, ensure the minter filter is updated by appropriate admin.
```

Comment:
(same callout)

Reply:
Yesser yesser – did do in the associated ticket tagging Dan. 😄